status.py

```diff
@@ -47,6 +48,7 @@ def is_server_error(code):
     204: 'HTTP_204_NO_CONTENT',
     205: 'HTTP_205_RESET_CONTENT',
     206: 'HTTP_206_PARTIAL_CONTENT',
+    207: 'HTTP_207_MULTI_STATUS',
```

Comment:
I would also add the following status_codes:
```suggestion
    207: 'HTTP_207_MULTI_STATUS',
```
208: 'HTTP_208_ALREADY_REPORTED',